packages/device_info_plus/device_info_plus/pubspec.yaml

```diff
@@ -5,6 +5,9 @@ version: 10.1.0
 homepage: https://plus.fluttercommunity.dev/
 repository: https://github.com/fluttercommunity/plus_plugins/tree/main/packages/device_info_plus/device_info_plus
 issue_tracker: https://github.com/fluttercommunity/plus_plugins/labels/device_info_plus
+topics:
+  - device
+  - information
```

Comment:
All other packages have `utils` topic but not this (and android alarm manager plus). Should this be added?

Reply:
Yes, missed that one. Thanks.